shm/inc/game.hpp

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+#include <memory>
+
+#include "map.hpp"
+#include "player.hpp"
+#include "time.hpp"
+#include "store.hpp"
```

Comment:
If we follow alphabetical order then store should be before time.

Reply:
Done.